routers/org/org_labels.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package org
+
+import (
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/auth"
+	"code.gitea.io/gitea/modules/context"
+)
+
+// RetrieveLabels find all the labels of an organization
+func RetrieveLabels(ctx *context.Context) {
+	labels, err := models.GetLabelsByOrgID(ctx.Org.Organization.ID, ctx.Query("sort"), models.ListOptions{})
+	if err != nil {
+		ctx.ServerError("RetrieveLabels.GetLabels", err)
+		return
+	}
+	for _, l := range labels {
+		l.CalOpenIssues()
+	}
+	ctx.Data["Labels"] = labels
+	ctx.Data["NumLabels"] = len(labels)
+	ctx.Data["SortType"] = ctx.Query("sort")
+}
+
+// NewLabel create new label for repository
+func NewLabel(ctx *context.Context, form auth.CreateLabelForm) {
+	ctx.Data["Title"] = ctx.Tr("repo.labels")
```

Comment:
```suggestion
	ctx.Data["Title"] = ctx.Tr("org.labels")
```

Reply:
I just reused the translation here as to not create a new one for each language to have to do (which seems to be what others have done when code is shared across areas). I could change it if needed

Reply:
Oh. It's confusing that there's a `org.labels` with a different usage. Maybe rename `org.labels` to something more specific? (and keep using `repo.labels` here).

Reply:
Fixed, renamed new var to more descriptive:
repo.org_labels_desc